crates/http/Cargo.toml

```diff
@@ -9,13 +9,14 @@ anyhow = { workspace = true }
 http = { workspace = true }
 http-body-util = { workspace = true }
 hyper = { workspace = true }
-indexmap = "1"
+indexmap = "2"
 percent-encoding = "2"
 routefinder = "0.5.4"
 serde = { workspace = true }
 spin-app = { path = "../app", optional = true }
 spin-locked-app = { path = "../locked-app" }
 tracing = { workspace = true }
+wasmtime = { workspace = true }
```

Comment:
This should, I think, be under the `runtime` feature.  We have consumers of this crate who use it for things like route parsing (e.g. the cloud plugin), but do not want to drag the whole of wasmtime along with it.

(I'm happy if you prefer to reorganise things to avoid the feature thing, but I really really want a way for cloud-plugin to get what it needs without including wasmtime.  Whether that is features or separate crates, I don't care, but it makes a big difference!)

Reply:
Ah, I just tried updating cloud-plugin and it looks like some of the other recebt changes have also brought in wasmtime dependency.  I'll get a PR out for all of it together.

Reply:
Okay, I did some more investigation and I'm gonna punt on it for now.  The wasmtime dependency makes 1 min of difference to a clean cloud-plugin build, but not enough difference to an incremental build to be worth the candle, and we are churning cloud-plugin dependencies a lot more slowly than when it drove me to extremes.  Something to maybe think about how we can streamline in a more sustainable way in future, but not worth fretting about for now.

Sorry for bending your ear @rylev!